### ISSUE TITLE
Better handling of large inputs / outputs for workflows controller?

### ISSUE THREAD

**Issue by dlebauer** [ebimodeling/ghgvc#71]
I have set up a kind of test harness for checking that any changes  i have made changes to the app/controllers/workflows_controller.rb

When I run  my version of the calculator on pecandev  thats ( http://www.ecosystemservicescalc.hmb ),
 I pick up the command line parameters that get_biome have been called  with ( ./ghgvc/log/development.log )

 Then  run the following command in a shell script

```sh 
 curl  --data-urlencode lat=52.64  --data-urlencode lng=-4.0 --data-urlencode id=1  --data-urlencode ecosystems="${ecosystems}" http://www.ecosystemservicescalc.hmb/get_biome
```

The problem is that the input "ecosystems" paramter is huge and the returned json file is equally huge

Could be doing something smarter or there a test harness already in place ?

**Comment by dlebauer** [comment]
@hmb1 are your changes on a particular branch?

**Comment by dlebauer** [comment]
@potterzot can you check if this is still an issue? If not, please close.

**Comment by potterzot** [comment]
I believe this is no longer an issue. Workflows_controller.rb doesn't do any direct netcdf loading. All loading happens in ghgvcR, and uses the new (ish) Saatchi.nc4 file. However, there are still other netcdf files that are separated. I don't know if the eventual intent is to combine all the netcdf files, but if it is, then that will have to happen first, and then the ghgvcR code will have to be updated to pull from the new file.

**Comment by dlebauer** [comment]
At this point, combining all files into one is lower priority than standardizing their format.
